src/main/java/seedu/task/logic/parser/AliasParser.java

```diff
@@ -0,0 +1,39 @@
+//@@author A0144939R
+package seedu.task.logic.parser;
+
+import static seedu.task.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.task.logic.commands.AliasCommand;
+import seedu.task.logic.commands.Command;
+import seedu.task.logic.commands.IncorrectCommand;
+
+public class AliasParser extends BaseParser {
+    
+    private final String FLAG_ALIAS_COMMAND = "aliasCommand";
+    private final String FLAG_ALIAS_VALUE ="aliasValue";
+    private final int FLAG_ALIAS_COMMAND_LENGTH = 2;
+
+    @Override
+    public Command parse(String command, String arguments) {
+        this.extractArguments(arguments);
+        if(getSingleKeywordArgValue(FLAG_ALIAS_COMMAND) != null && getSingleKeywordArgValue(FLAG_ALIAS_VALUE) != null) {
```

Comment:
`getSingleKeywordArgValue(FLAG_ALIAS_COMMAND)` can be set as a variable, since you use it again?